docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  session-handler:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: session-handler-v1.01
```

Comment:
can you rename your images to be session-handler:1.0 when you include the colon symbol you are using Docker's instead way to tag/version images.